src/DynamoManipulation/TranslationGizmo.cs

```diff
@@ -211,7 +211,7 @@ private object HitTest(Point source, Vector direction)
             using (var ray = GetRayGeometry(source, direction))
             {
                 //First hit test for position
-                if (ray.DistanceTo(ManipulatorOrigin) < tolerance)
+                if (ray.DistanceTo(Origin) < tolerance)
```

Comment:
The hit test should be w.r.t. the gizmo origin and not the original point location